job-descriptions/software-engineer-developer-productivity.md

```diff
@@ -0,0 +1,53 @@
+![logo](https://sourcegraph.com/.assets/img/sourcegraph-light-head-logo.svg)
+
+# Software Engineer - Developer Productivity
+
+We are looking for a software engineer to own developer productivity at Sourcegraph. You will be responsible for proactively identifying pain points in our developer workflows (by soliciting feeback from our dev team), and then quickly iterating to solve those issues (e.g. make CI faster and more reliable, automate more of our release process, implement GitHub and/or Slack bots to make common tasks easier for devs). As our needs grow over time, there is a potential for you to grow into a leadership role for a team that owns this area.
+
+## Qualifications
+
+We are looking for candidates with the following skills, and we will test for these things in our interview process:
+
+<!-- This section is WIP and I am looking for suggestions -->
+
+- Experience or willing to learn Go and TypeScript.
+- Experience configuring and using Continuous Integration systems (e.g. Buildkite, Travis).
+- Ability to communicate clearly and empathetically, especially in writing and documentation.
```

Comment:
I think Docker and shell scripting should be required. Also, excellent technical writing (i.e. documentation).